stable/firmament-poseidon/templates/firmament_deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "firmament.fullname" .}}
+  namespace: {{ .Values.namespace }}
```

Comment:
Why do you need the namespace here? It can be passed to the Helm CLI when installing the chart.